README.md

```diff
@@ -8,10 +8,22 @@
 This package provides basic interaction with a Clojure subprocess (REPL).
 It's based on ideas from the popular `inferior-lisp` package.
 
-`inf-clojure` has two components - a nice Clojure REPL with
-auto-completion and a minor mode (`inf-clojure-minor-mode`), which
-extends `clojure-mode` with commands to evaluate forms directly in the
-REPL.
+`inf-clojure` aims to expose the extensive self-documenting features
+of Clojure repls into an emacs package. Inf-clojure does not require
+middleware or complicated machinery, but it more a predefined set of
+commands for a repl. Because of this, you can interact with many more
+repl types than an nREPL-based tool like CIDER. Inf-clojure supports
+the following repls:
+- Clojure
+- clojurescript through cljs.main (no piggieback required)
```

Comment:
I don't think the clarification here is needed, as without nREPL obviously you don't need piggieback (at least it seems obvious to me). Might a good idea to capitalize all names consistently in this list.